phpunit/class-wp-theme-json-resolver-test.php

```diff
@@ -760,6 +760,20 @@ public function test_get_user_data_from_wp_global_styles_filter_state() {
 		$this->assertSameSets( array(), $post2 );
 	}
 
+	/**
+	 * @covers WP_Theme_JSON_Resolver_Gutenberg::get_user_data_from_wp_global_styles
+	 */
+	public function test_get_user_data_from_wp_global_styles_returns_created_post() {
+		// Switch to a theme that does have support.
+		switch_theme( 'block-theme' );
```

Comment:
```suggestion
		switch_theme( 'block-theme' );
		wp_set_current_user( self::$administrator_id );
```

With the correct capabilities to assign terms, this test passes.